docs/DeveloperGuide.md

```diff
@@ -1000,3 +1000,24 @@ testers are expected to do more *exploratory* testing.
 similar to the test cases found in `Editing Seller Details`.
 
 </box>
+
+### B.2 Add Seller/Buyer command
+
+#### B.2.1 Motivation
+- The `addSeller` and `addBuyer` commands currently identify individuals within the `Persons` list using `name` as a unique identifier. There are two main reasons for this approach.
+- Firstly, we have been using the `name` to identify `Buyers` and `Sellers` in various commands, such as adding or deleting a house. 
+- Secondly, to distinguish between individuals sharing the same `name`, we append numbers to their names, resulting in unique identifiers like `John Doe 1` and `John Doe 2`, which is why we allow alphanumeric characters in the `name` field.
+- However, this method has led to confusion, as relying on `name` as unique identifiers is problematic due to the common occurrence of name duplication.
+
+#### B.2.2 Implementation
+- To address this issue, we are considering a change to use more distinct identifiers such as `email` or `phone` numbers, in conjunction with names, as these fields are highly unlikely to be identical between two individuals, thereby improving the clarity and reliability of our identification process.
```

Comment:
Actually are we gonna having this? I think sometimes there are cases that people share phone numbers or even email.

Reply:
To be honest, I get what you mean, but I think that don't have to think so far? Because I do not think we have any other fields that are uniquely identifiable besides email and phone. And the tester comment given was to use either email or phone to uniquely identify the person